### PR TITLE
Display the role of the governing body in secretaries ( "Діловодства" )tab  

### PR DESCRIPTION
**Pull request by christykobyletska**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: EPlast/EPlast.BLL/Services/GoverningBodies/GoverningBodiesService.cs [hunk 1]
```diff
@@ -193,5 +193,38 @@ public async Task<GoverningBodyProfileDTO> GetGoverningBodyDocumentsAsync(int go
         {
             return await _securityModel.GetUserAccessAsync(userId);
         }
+
+        public async Task<IEnumerable<GoverningBodyAdministrationDTO>> GetAdministrationsOfUserAsync(string UserId)
+        {
+            var admins = await _repoWrapper.GoverningBodyAdministration.GetAllAsync(a => a.UserId == UserId && (a.EndDate > DateTime.Now || a.EndDate == null),
```
Comment (sharix4) [inline EPlast/EPlast.BLL/Services/GoverningBodies/GoverningBodiesService.cs:199]: _repoWrapper.GoverningBodyAdministration.GetAllAsync(a => a.UserId == UserId &&  && c.Status
we have administrator status in the database and you can find all valid administrators using this status && c.Status,

File: EPlast/EPlast.BLL/Services/GoverningBodies/GoverningBodiesService.cs [hunk 2]
```diff
@@ -193,5 +193,38 @@ public async Task<GoverningBodyProfileDTO> GetGoverningBodyDocumentsAsync(int go
...
+        public async Task<IEnumerable<GoverningBodyAdministrationDTO>> GetPreviousAdministrationsOfUserAsync(string UserId)
+        {
+            var admins = await _repoWrapper.GoverningBodyAdministration.GetAllAsync(a => a.UserId == UserId && a.EndDate < DateTime.Now,
+                 include:
+                 source => source.Include(c => c.User).Include(c => c.AdminType).Include(a => a.GoverningBody)
+                 );
+
+            foreach (var admin in admins)
+            {
+                admin.GoverningBody.GoverningBodyAdministration = null;
```
Comment (sharix4) [inline EPlast/EPlast.BLL/Services/GoverningBodies/GoverningBodiesService.cs:224]: and why do it?

File: EPlast/EPlast.Tests/Controllers/GoverningBodiesControllerTests.cs [hunk 3]
```diff
@@ -309,7 +309,7 @@ public async Task RemoveAdmin_Valid_Test()
                 .ReturnsAsync(new GoverningBodyAdministrationDTO());
 
             // Act
-            var result = await _controller.RemoveAdmin(FakeId);
+            var result = await _governingBodiesController.RemoveAdmin(FakeId);
```
Comment (sharix4) [inline EPlast/EPlast.Tests/Controllers/GoverningBodiesControllerTests.cs:312]: FakeId --what is this name?

**Review by sharix4 — CHANGES_REQUESTED** [review]

File: EPlast/EPlast.Tests/Controllers/GoverningBodiesControllerTests.cs (hunk 3 above)
Comment (christykobyletska) [reply EPlast/EPlast.Tests/Controllers/GoverningBodiesControllerTests.cs:312]: just Id for test

Comment (sharix4) [reply EPlast/EPlast.Tests/Controllers/GoverningBodiesControllerTests.cs:312]: Ok

File: EPlast/EPlast.BLL/Services/GoverningBodies/GoverningBodiesService.cs (hunk 1 above)
Comment (christykobyletska) [reply EPlast/EPlast.BLL/Services/GoverningBodies/GoverningBodiesService.cs]: done

**Review by sharix4 — APPROVED** [review]

**Review by nicoolle — APPROVED** [review]

**Review by Vinnytile — APPROVED** [review]

**Pull request merged**